src/Map/Map.js

```diff
@@ -27,15 +27,12 @@ class Map extends React.Component {
     // this is used to create a unique identifier for the map div
     this.target = `_ol_kit_map_${nanoid(6)}`
 
-    // if no map was passed, create the map
-    this.map = !props.map ? createMap({ target: this.target }) : props.map
```

Comment:
this being in the constructor caused a hidden problem until the browser actually "rendered" a map that was just white